src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1027,7 +1027,7 @@ private Symbol UnwrapAlias(Symbol symbol, out AliasSymbol alias, DiagnosticBag d
                     diagnostics);
             }
 
-            if (options.IsAttributeTypeLookup())
+            if (Compilation.LanguageVersion < LanguageVersion.CSharp8 && options.IsAttributeTypeLookup())
```

Comment:
>Compilation.LanguageVersion < LanguageVersion.CSharp8 [](start = 16, length = 53)

Should be `Compilation.LanguageVersion.AllowGenericAttributes()`

Reply:
Done